addons/pinniped/config-controller/hack/run.sh

```diff
@@ -5,22 +5,44 @@
 
 set -euo pipefail
 
-MANIFESTS_DIR="./manifests"
-RBAC_FILE="${MANIFESTS_DIR}/role.yaml"
-DEPLOYMENT_FILE="${MANIFESTS_DIR}/deployment.yaml"
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TF_ROOT="${MY_DIR}/../../../.."
+
+# location for the package install
+MANAGEMENT_DIR="${TF_ROOT}/packages/management"
+TANZU_AUTH_DIR="${MANAGEMENT_DIR}/tanzu-auth"
+TANZU_AUTH_BUNDLE_DIR="${TANZU_AUTH_DIR}/bundle"
+TANZU_AUTH_CONFIG_DIR="${TANZU_AUTH_BUNDLE_DIR}/config"
+TANZU_AUTH_PACKAGE="${TANZU_AUTH_CONFIG_DIR}/package.yaml"
+
+# location to deploy the image
+REGISTRY_NAME="harbor-repo.vmware.com"
+REGISTRY_PROJECT="tkgiam"
+REGISTRY="${REGISTRY_NAME}/${REGISTRY_PROJECT}"
+
+# name of image to match that defined in /packages/management/tanzu-auth
+DEPLOYMENT_NAME="tanzu-auth-controller-manager"
+NAMESPACE_NAME="tanzu-auth"
 
 # Always run from config-controller directory for reproducibility
 cd "${MY_DIR}/.."
 
 tag="dev"
 # tag="$(uuidgen)" # Uncomment to create random image every time
-image="harbor-repo.vmware.com/tkgiam/$(whoami)/pinniped-config-controller-manager:$tag"
+image="${REGISTRY}/$(whoami)/${DEPLOYMENT_NAME}:$tag"
 
+# generate new RBAC into /packages/management/tanzu-auth/bundle/config
 ./hack/generate.sh
-kubectl apply -f "${RBAC_FILE}"
+
+# the namespace must exist before the package can be deployed
+kubectl apply -f "${TANZU_AUTH_CONFIG_DIR}/namespace.yaml"
```

Comment:
suggestion: did you consider deploying the package by actually building it (`imgpkg push`) and deploying it (with a `PackageInstall`)? i think that would be exercise the package with more fidelity

Reply:
coming in next push.